tacotron2/model.py

```diff
@@ -481,6 +481,7 @@ def parse_batch(self, batch):
         output_lengths = to_gpu(output_lengths).long()
 
         return (
+            mel_padded,
```

Comment:
any reason for this when we're already returning it as part of tuple in line 486? just for convenience?